README.rst

```diff
@@ -335,6 +335,23 @@ The following flags are accepted:
 | Determines the proto option Gazelle uses to group .proto files into rules                        |
 | when in ``package`` mode. See details in `Directives`_ below.                                    |
 +--------------------------------------------------------------+-----------------------------------+
+| :flag:`-proto_strip_import_prefix example.com/repo`          |                                   |
```

Comment:
Nit: change the value from `example.com/repo` to `vendor`. I think that will be the most common use for this flag.